__tests__/hooks/useSendBox.js

```diff
@@ -0,0 +1,65 @@
+import { timeouts } from '../constants.json';
+
+import negate from '../setup/conditions/negate';
+import speechRecognitionStartCalled from '../setup/conditions/speechRecognitionStartCalled';
+import speechSynthesisUtterancePended from '../setup/conditions/speechSynthesisUtterancePended';
+import uiConnected from '../setup/conditions/uiConnected';
+
+// selenium-webdriver API doc:
+// https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html
+
+jest.setTimeout(timeouts.test);
+
+test('sendBoxDictationStarted should return if dictation is started or not', async () => {
```

Comment:
Does this test `useSendBox` or `useSendBoxDictationStarted`? Might make sense to rename the file.